src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -80,7 +80,7 @@ const PAGES_UNIT_NAME = 'pages';
 const PDFJS_TEXT_LAYER_MODE = {
     DISABLE: 0, // Should match TextLayerMode enum in pdf_viewer.js
     ENABLE: 1,
-    ENABLE_ENHANCE: 2,
+    ENABLE_PERMISSIONS: 2,
```

Comment:
Do we use this value anywhere? If not do we even need this?

Reply:
I think if we're going to retrofit, we may need to add an additional ENUM for `ENABLE_ENHANCE`, so we may want to keep `ENABLE_PERMISSIONS` there as 2 and have `ENABLE_ENHANCE` at 3 if that make sense. That way we don't have to modify the logic to replace PERMISSIONS.